2022-01-31 11:48:24,975 INFO    Thread-2  :364 [wandb_setup.py:_flush():71] setting env: {'root_dir': '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10', 'entity': 'paulina', 'project': 'ImageClassifierCIFAR10', 'run_id': '43t1jqjq', 'sweep_id': 'mjc1k0xo', 'sweep_param_path': '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/sweep-mjc1k0xo/config-43t1jqjq.yaml'}
2022-01-31 11:48:24,976 WARNING Thread-2  :364 [wandb_setup.py:_flush():71] could not save program above cwd: /home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/ray/workers/default_worker.py
2022-01-31 11:48:24,976 INFO    Thread-2  :364 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-31 11:48:24,976 INFO    Thread-2  :364 [wandb_init.py:_log_setup():371] Logging user logs to /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_114824-43t1jqjq/logs/debug.log
2022-01-31 11:48:24,976 INFO    Thread-2  :364 [wandb_init.py:_log_setup():372] Logging internal logs to /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_114824-43t1jqjq/logs/debug-internal.log
2022-01-31 11:48:24,976 INFO    Thread-2  :364 [wandb_init.py:init():404] calling init triggers
2022-01-31 11:48:24,977 INFO    Thread-2  :364 [wandb_init.py:init():409] wandb.init called with sweep_config: {'learning_rate': 0.0007524197932810306, 'optimizer': 'adam'}
config: {}
2022-01-31 11:48:24,977 INFO    Thread-2  :364 [wandb_init.py:init():460] starting backend
2022-01-31 11:48:24,977 INFO    Thread-2  :364 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-31 11:48:24,993 INFO    Thread-2  :364 [backend.py:ensure_launched():216] starting backend process...
2022-01-31 11:48:25,003 INFO    Thread-2  :364 [backend.py:ensure_launched():221] started backend process with pid: 1448
2022-01-31 11:48:25,004 INFO    Thread-2  :364 [wandb_init.py:init():469] backend started and connected
2022-01-31 11:48:25,006 INFO    Thread-2  :364 [wandb_run.py:_config_callback():956] config_cb None None {'learning_rate': 0.0007524197932810306, 'optimizer': 'adam'}
2022-01-31 11:48:25,008 INFO    Thread-2  :364 [wandb_init.py:init():533] updated telemetry
2022-01-31 11:48:25,009 INFO    Thread-2  :364 [wandb_init.py:init():563] communicating current version
2022-01-31 11:48:26,364 INFO    Thread-2  :364 [wandb_init.py:init():568] got version response 
2022-01-31 11:48:26,364 INFO    Thread-2  :364 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-31 11:48:26,519 INFO    Thread-2  :364 [wandb_init.py:init():606] starting run threads in backend
2022-01-31 11:48:28,719 INFO    Thread-2  :364 [wandb_run.py:_console_start():1810] atexit reg
2022-01-31 11:48:28,721 INFO    Thread-2  :364 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-31 11:48:28,721 INFO    Thread-2  :364 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-31 11:48:28,721 ERROR   Thread-2  :364 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread of the main interpreter
2022-01-31 11:48:28,724 INFO    Thread-2  :364 [wandb_init.py:init():633] run started, returning control to user process
2022-01-31 11:54:05,433 INFO    MainThread:364 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-31 11:54:05,436 INFO    MainThread:364 [wandb_run.py:_restore():1752] restore
2022-01-31 11:54:05,440 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1079
  total_bytes: 1079
}

2022-01-31 11:54:05,630 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1079
  total_bytes: 1079
}

2022-01-31 11:54:05,834 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1079
  total_bytes: 8197
}

2022-01-31 11:54:05,935 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1079
  total_bytes: 8197
}

2022-01-31 11:54:06,037 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8197
  total_bytes: 8197
}

2022-01-31 11:54:06,138 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8197
  total_bytes: 8197
}

2022-01-31 11:54:06,239 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8197
  total_bytes: 8197
}

2022-01-31 11:54:06,340 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8197
  total_bytes: 8197
}

2022-01-31 11:54:06,442 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8197
  total_bytes: 8197
}

2022-01-31 11:54:06,543 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8197
  total_bytes: 8197
}

2022-01-31 11:54:06,896 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8197
  total_bytes: 8197
}

2022-01-31 11:54:07,221 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8197
  total_bytes: 8197
}
local_info {
}

2022-01-31 11:54:08,312 INFO    MainThread:364 [wandb_run.py:_append_history():2130] rendering history
2022-01-31 11:54:08,312 INFO    MainThread:364 [wandb_run.py:_append_summary():2085] rendering summary
2022-01-31 11:54:08,312 INFO    MainThread:364 [wandb_run.py:_append_files():2180] logging synced files
